packages/react-components/src/field/src/FieldContext.tsx

```diff
@@ -0,0 +1,160 @@
+import { ElementType, ReactNode, createContext, useContext } from "react";
+import { cssModule, normalizeSize } from "../../shared";
+import { isNil } from "lodash";
+
+export interface CommonFieldContextType {
```

Comment:
Do you have to export a base interface like this one?

Reply:
we seem to be using it in the useField typings

Reply:
Removed that interface, not an issue anymore